zipline/finance/risk/period.py

```diff
@@ -56,14 +56,14 @@ def __init__(self, start_date, end_date, returns, trading_schedule,
 
         self.start_date = start_date
         self.end_date = end_date
-        self.trading_schedule = trading_schedule
+        self.trading_calendar = trading_calendar
 
-        trading_dates = trading_schedule.trading_dates(
-            start=self.start_date,
-            end=self.end_date,
+        trading_sessions = trading_calendar.sessions_in_range(
+            self.start_date,
```

Comment:
`start_date` and `end_date` are actually midnight timestamps (ie, sessions).  I've renamed all the relevant parameters and variables to be clearer.